CONTRIBUTING.rst

```diff
@@ -1,461 +1,465 @@
-============
-Contributing
-============
-
-Contributions are welcome, and they are greatly appreciated! Every
-little bit helps, and credit will always be given. 
-
-.. toctree::
-   :numbered:
-   :maxdepth: 2
-
-   types_of_contributions
-   contributor_setup
-   contributor_guidelines
-   contributor_testing
-   core_committer_guide
-
-
-Types of Contributions
-----------------------
-
-You can contribute in many ways:
-
-Create Cookiecutter Templates
-~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
-
-Some other Cookiecutter templates to list in the :ref:`README <readme>` would
-be great.
-
-If you create a Cookiecutter template, submit a pull request adding it to
-README.rst.
-
-Report Bugs
-~~~~~~~~~~~
-
-Report bugs at https://github.com/audreyr/cookiecutter/issues.
-
-If you are reporting a bug, please include:
-
-* Your operating system name and version.
-* Any details about your local setup that might be helpful in troubleshooting.
-* If you can, provide detailed steps to reproduce the bug.
-* If you don't have steps to reproduce the bug, just note your observations in
-  as much detail as you can. Questions to start a discussion about the issue
-  are welcome.
-
-Fix Bugs
-~~~~~~~~
-
-Look through the GitHub issues for bugs. Anything tagged with "bug"
-is open to whoever wants to implement it.
-
-Implement Features
-~~~~~~~~~~~~~~~~~~
-
-Look through the GitHub issues for features. Anything tagged with "enhancement"
-and "please-help" is open to whoever wants to implement it.
-
-Please do not combine multiple feature enhancements into a single pull request.
-
-Note: this project is very conservative, so new features that aren't tagged
-with "please-help" might not get into core. We're trying to keep the code base
-small, extensible, and streamlined. Whenever possible, it's best to try and 
-implement feature ideas as separate projects outside of the core codebase.

```

Comment:
It looks like this text was updated directly in `CONTRIBUTING.rst` and not in `docs/types_of_contributions.rst`, could you please fix this?